gulpfile.js

```diff
@@ -20,6 +20,8 @@ var collapse = require('bundle-collapser/plugin');
 var argv  = require('yargs').argv
 var path = require('path');
 var package = require('./package.json');
+var fs = require('fs');
```

Comment:
[nit] can we move `var fs` before `var package` since the later one is a local file.

Reply:
yes
while i'm at it, is there any logic in the order for the non local dependencies ? alphabetical, date of addition ?

Reply:
No guideline for the order because some require() may need other ones. I'm used to gather related dependencies (e.g. `gulp-*`) and declare the `package.json` at the end.